analyzer/codechecker_analyzer/cmd/analyzers.py

```diff
@@ -119,8 +119,13 @@ def main(args):
         if args.dump_config == 'clang-tidy':
             subprocess.call([binary, '-dump-config', '-checks=*'])
         elif args.dump_config == 'clangsa':
-            # TODO: Not supported by ClangSA yet!
-            LOG.warning("'--dump-config clangsa' is not supported yet.")
+            ret = subprocess.call([binary, '-cc1',
+                                   '-analyzer-checker-option-help'],
+                                  stderr=subprocess.PIPE)
+
+            if ret:
+                # TODO: Not supported by ClangSA yet!
```

Comment:
Please remove the TODO comment from here and add a comment that older clang versions does not support this flag. clang9 is the first?

Reply:
Should be.